src/process-events.c

```diff
@@ -367,10 +397,24 @@ Pwd:;
     SET_OFFSET(CRC_DENTRY_D_PARENT);
     u32 parent = *(u32 *)offset; // offset of d_parent
 
-    SEND_PATH_N(9);
-    bpf_tail_call(ctx, &tail_call_table, SYS_EXECVE_4_8);
+    u64 _to_skip = (u64)bpf_map_lookup_elem(&read_path_skip, &to_skip);
+    if (_to_skip) {
+        __builtin_memcpy(&to_skip, (void*)_to_skip, sizeof(u64));
+        SKIP_PATH_N(100);
```

Comment:
whats the reasoning behind these SEND_PATH magic values?

Reply:
This number (I bumped it up to `150`) is determined experimentally, keeping us under `4096` instructions. Sending takes up far more instructions than skipping, so this felt like a good balance.